src/components/v1/V1Project/V1PayButton.tsx

```diff
@@ -52,63 +53,51 @@ export default function V1PayButton({
   const overridePayDisabled =
     projectId && disablePayOverrides[readNetwork.name]?.has(projectId)
 
+  const shouldDisableButton =
```

Comment:
Refactor of the disabled state. Instead of returning early, we can return once, but define the `visible` prop of `tooltip` accordingly.